assets/style.styl

```diff
@@ -223,13 +238,14 @@ for $name, $motion in $animBank
     animation-name fadeIn- + $name
     animation-duration $animDuration
     animation-timing-function $animEasingIn
+    animation-delay 20ms // delay for calculating arrow positions
```

Comment:
Delay for `getBoundingClientRect()` to get correct initial values - if we animate instantly, we get the tween values which can produce incorrect calculations later, depending on UI speed.